src/modules/Home.service.js

```diff
@@ -21,6 +21,21 @@ const HomePage = async () => {
     movieLists.innerHTML = renderMovieHandler(movieListData);
   }
   pagination.innerHTML = renderPaginationHandler(movieListData);
+  // - TODO: attach event listner to Like button
+
+  const likeButtons = document.querySelectorAll('.material-symbols-outlined');
+  likeButtons.forEach((likeButton) => {
+    likeButton.addEventListener('click', (event) => {
+      likeButton.classList.add('red');
+      // - TODO: also increment in the UI
+      postLike(event.target.id)
+        .then(() => {
+          window.location.reload();
+        });
```

Comment:
Could you avoid reloading the page when a user likes an item? Updating the number of likes of the item on the DOM alone can be an option. This will likely give a better user experience and will be faster than reloading the whole page.

Reply:
@emmiiorji  good call !!